src/containers/ProfileDetail/StudyTab/StudyTab.tsx

```diff
@@ -0,0 +1,55 @@
+import { useEffect, useState } from 'react';
+import { useInterSectionObserver } from '@src/hooks/useIntersectionObserver';
+import { StudyListType } from '@interfaces/studyList';
+import { StudyList } from '@components';
+import { getStudyList } from '@apis/studyGroups';
+
+function StudyTab() {
+  const [studyList, setStudyList] = useState<StudyListType>([]);
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState(false);
+  const [hasNext, setHasNext] = useState(true);
+  const [lastStudyId, setLastStudyId] = useState(0);
+  const { targetRef } = useInterSectionObserver({
+    onTargetObserve: () => {
+      if (lastStudyId < 2) setLastStudyId(lastStudyId + 1);
+    },
+    observerOptions: {},
+  });
+
+  useEffect(() => {
+    if (!hasNext) return;
+    (async () => {
+      try {
+        setLoading(true);
+        const { content: newStudylist, hasNext: newHasNext } =
```

Comment:
```suggestion
        const { content: newStudyList, hasNext: newHasNext } =
```
큰 문제는 아니지만 Camel Case 통일을 위해 newStudyList로 적으면 좋을 것 같습니다!